reposerver/repository/repository.go

```diff
@@ -633,6 +633,8 @@ func runConfigManagementPlugin(appPath string, envVars *v1alpha1.Env, q *apiclie
 		env = append(env, environ...)
 	}
 	env = append(env, q.ApplicationSource.Plugin.Env.Environ()...)
+	env = append(env, "KUBEVERSION="+q.KubeVersion)
```

Comment:
To be consistent please use `KUBE_VERSION` instead of `KUBEVERSION`

Reply:
:+1: Fixed. Also added test verification of KUBE_API_VERSION.